components/dash-core-components/src/components/Dropdown.react.js

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React, {Component, lazy, Suspense} from 'react';
 import dropdown from '../utils/LazyLoader/dropdown';
+import './css/Dropdown.css';
```

Comment:
This file is already imported here: https://github.com/plotly/dash/blob/b7f550f1b8a48e904af8c7c837506ecfaf98c14a/components/dash-core-components/src/fragments/Dropdown.react.js#L7